axelrod/_strategy_utils.py

```diff
@@ -45,11 +45,19 @@ def limited_simulate_play(player_1, player_2, h1):
     player_2's strategy method to set any internal variables as needed. If you
     need a more complete simulation, see `simulate_play` in player.py. This
     function is specifically designed for the needs of MindReader."""
-    h2 = player_2.strategy(player_1)
+    h2 = cheating_bastard(player_1, player_2)
     update_history(player_1, h1)
     update_history(player_2, h2)
 
 
+def cheating_bastard(cheater, opponent):
```

Comment:
This function name will need to change. Similarly `cheater` needs to be something more verbose, I've been using the word `cheater` as shorthand, but technically it's just that it does not obey Axelrod's original rules.

I suggest:

```python
def simulated_strategy(player, opponent):
    ```
    Returns the simulated strategy of a player against an opponent. If the strategy has an implemented `simulation_strategy` then that gets called. 
    ```
```